pkgs/top-level/python-packages.nix

```
@@ -7823,12 +7823,22 @@ in {
 
   pyqt-builder = callPackage ../development/python-modules/pyqt-builder { };
 
+  pyqt6-builder = callPackage ../development/python-modules/pyqt-builder/6.x.nix { };
+
   pyqt4 = callPackage ../development/python-modules/pyqt/4.x.nix { };
 
   pyqt5 = callPackage ../development/python-modules/pyqt/5.x.nix { };
 
+  pyqt6 = callPackage ../development/python-modules/pyqt/6.x.nix { };
```

```suggestion
  pyqt4 = callPackage ../development/python-modules/pyqt/4.nix { };

  pyqt5 = callPackage ../development/python-modules/pyqt/5.nix { };

  pyqt6 = callPackage ../development/python-modules/pyqt/6.nix { };
```

I know this is a bit unrelated but the shorter name without x is nicer.